securedrop_client/logic.py

```diff
@@ -413,6 +415,15 @@ def __init__(  # type: ignore [no-untyped-def]
         ):
             os.chmod(self.last_sync_filepath, 0o600)
 
+    def on_main_queue_updated(self, num_items: int) -> None:
```

Comment:
```suggestion
    @pyqtSlot(int)
    def _on_main_queue_updated(self, num_items: int) -> None:
```

Decorating [slots][doc] is [mostly conventional][so] but we're starting to do it systematically. You can import `pyqtSlot` from `PyQt5.QtCore` if it isn't already. The decoration has the nice advantage of expressing the intended use clearly, and helps keeping in mind that nothing too time-consuming should be happening in the method.

  [so]: https://stackoverflow.com/a/40330912
  [doc]: https://doc.qt.io/qt-5.15/signalsandslots.html
  
**Nitpicking**: The method could be private. Since it's new, we know it's not being used anywhere else, and this might be the best time to make that explicit!

Reply:
Thanks, I wasn't aware of this decorator!